src/main/kotlin/io/openfuture/snapshot/runner/CommandLineSnapshotRunner.kt

```diff
@@ -3,37 +3,68 @@ package io.openfuture.snapshot.runner
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.*
 import com.github.ajalt.clikt.parameters.types.int
-import io.openfuture.snapshot.dto.ExportSnapshotRequest
-import io.openfuture.snapshot.exporter.SnapshotExporter
+import io.openfuture.snapshot.dto.SnapshotRequest
+import io.openfuture.snapshot.exporter.CsvSnapshotExporter
+import io.openfuture.snapshot.snapshoter.SnapshotFactory
+import io.openfuture.snapshot.snapshoter.SnapshotMode
+import io.openfuture.snapshot.util.awaitAll
+import java.util.concurrent.Executors
 
-class CommandLineSnapshotRunner(private val snapshotExporter: SnapshotExporter) : CliktCommand(name = "snapshot") {
+class CommandLineSnapshotRunner() : CliktCommand(name = "snapshot") {
 
     private val contractAddress: String? by option(help = "Address of token Smart contract").required()
     private val decimals: Int by option(help = "Snapshot decimals value").int().default(8)
```

Comment:
We agreed that drom and to parameters would be optional, but it's marked as required

Reply:
sorry did not notice(, will be changed from another branch and PR